plugins/package/README.md

```diff
@@ -13,6 +13,21 @@ keytool -genkey -alias "signing key 2" -keystore signingkeys.pfx -storepass "key
 # To trust freetsa.org, download their CA cert and import into keystore
 keytool -import -alias "freetsa" -keystore signingkeys.pfx -storepass "keystore password" -file ~/Downloads/cacert.pem
 
+# Build a CPB
+./corda-cli.sh package create-cpb \
+    --cpks "mycpk0.cpk mycpk1.cpk" \
```

Comment:
Does it work without the quotes? We want tab completion to help users enter the filenames. We might need to look at how the parameter is configured if we need the quotes.

Reply:
It did not work without the quotes. Good point thanks! Let me see if I can make it work without the quotes.

Reply:
Done - passed in cpks are made `Parameters` to allow auto-completion-.